docs/blog-post.md

```diff
@@ -0,0 +1,286 @@
+<h1><b>How to make akka serialization bulletproof</b></h1>
+
+![bulletproof shield](https://images.unsplash.com/photo-1561156772-a44477f220a5?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1974&q=80)
+<i>bulletproof shield from [Unsplash](https://images.unsplash.com/photo-1561156772-a44477f220a5) by [NIFTYART](https://unsplash.com/@niftyartofficial1_)</i>
+
+Every message leaving a JVM boundary in Akka needs to be serialized first. However, the existing solutions for serialization in Scala leave a lot of room for runtime failures that are NOT reported in compile time. We’re glad to introduce [Akka Serialization Helper](https://github.com/VirtusLab/akka-serialization-helper), a toolkit including a Circe-based, runtime-safe serializer and a set of Scala compiler plugins to counteract the common caveats when working with Akka serialization.
```

Comment:
I'd use **bold** rather than CAPS for stressing `not`. Caps are more suited for non-formatted text, like in comments etc.

Reply:
Done